### ISSUE TITLE
The entire thing is uh.. broken..

### ISSUE THREAD

**Issue by Noopch** [DiscordStyles/Slate#135]
### Describe the bug

literally nothing but the fonts work

### To Reproduce

literally just install it

### Screenshots

![image](https://github.com/DiscordStyles/Slate/assets/110434808/e4c0fb88-8869-4437-ba26-2bd72ed06a52)

### Information

**Discord channel**: _Stable/PTB/Canary_  
**OS**: Windows
**Mod**: Vencord
**Discord language**: English

**Comment by Gibbu** [comment]
Yep. Slate was a very very BIG theme and when Discord did their whole convert from Babel to SWC, it changed way too many things.
I planned on doing a rewrite ages ago, but then life happened, discord did more big updates that broke the theme even more.
All adding up to me just not being bothered.

Sorry about that :\

**Comment by sorae42** [comment]
> Yep. Slate was a very very BIG theme and when Discord did their whole convert from Babel to SWC, it changed way too many things. I planned on doing a rewrite ages ago, but then life happened, discord did more big updates that broke the theme even more. All adding up to me just not being bothered.
> 
> I don't really plan on fixing it any time soon as the themes that are currently fixed is enough for me to not go crazy.
> 
> Sorry about that :\

You should definitely pin this announcement up and update the readme so people wouldn't make anymore issue like this.

**Comment by Noopch** [comment]
> > Yep. Slate was a very very BIG theme and when Discord did their whole convert from Babel to SWC, it changed way too many things. I planned on doing a rewrite ages ago, but then life happened, discord did more big updates that broke the theme even more. All adding up to me just not being bothered.
> > I don't really plan on fixing it any time soon as the themes that are currently fixed is enough for me to not go crazy.
> > Sorry about that :\
> 
> You should definitely pin this announcement up and update the readme so people wouldn't make anymore issue like this.

yeah
